graphsdk/src/androidTest/java/com/microsoft/graph/functional/OutlookTests.java

```diff
@@ -41,4 +38,22 @@ public void testSendMail() {
         message.toRecipients = recipients;
         testBase.graphClient.getMe().getSendMail(message, true).buildRequest().post();
     }
+
+    public void testGetFindMeetingTimes() {
+        TestBase testBase = new TestBase();
+        List<AttendeeBase> attendees = new ArrayList<>();
+        AttendeeBase attendeeBase = new AttendeeBase();
+        EmailAddress email = new EmailAddress();
+        email.address = "katiej@mod810997.onmicrosoft.com";
```

Comment:
Should we have a constants file to handle mock data? Would it make it easier to update all of these in the future?